addons/web_tip/static/src/js/tip.js

```diff
@@ -250,6 +250,10 @@ var Tip = Class.extend({
                     def.resolve();
                 }
             });
+            self.$element.on('mousedown', function($ev) {
+                self.end_tip(tip);
+                def.resolve();
+            });
```

Comment:
Not sure this code is needed, but sure that you should probably update tip `crm_tip_4` and make it work like `project_tip_2`